plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -41,12 +42,23 @@ export type EntitiesCatalog = {
 // Locations
 //
 
+export type status = {
+  timestamp: DatabaseLocationUpdateLogEvent['created_at'] | null;
+  status: DatabaseLocationUpdateLogEvent['status'] | null;
+  message: DatabaseLocationUpdateLogEvent['message'] | null;
+};
```

Comment:
just write out the types - and besides, these should be `| undefined` shouldn't they? so a conversion needs to happen either way. Same for timestamps, that should be guaranteed to be iso8601/rfc3339 externally

Reply:
- [x] write out the types explicitly